prometheus-rules/templates/prometheusRule.yaml

```diff
@@ -0,0 +1,11 @@
+{{- range .Values.prometheusRule}}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ .name }}
+spec:
+  groups:
+{{ toYaml .groups | indent 4 }}
+---
+{{- end }}
```

Comment:
move the rules from values.yaml into templates like prometheusRule.solr.yaml, prometheusRule.k8s.yaml.